binder/binder.c

```diff
@@ -4109,10 +4109,12 @@ static int binder_wait_for_work(struct binder_thread *thread,
 	struct binder_proc *proc = thread->proc;
 	int ret = 0;
 
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(6,0,0)
 	freezer_do_not_count();
+#endif
 	binder_inner_proc_lock(proc);
 	for (;;) {
-		prepare_to_wait(&thread->wait, &wait, TASK_INTERRUPTIBLE);
+		prepare_to_wait(&thread->wait, &wait, TASK_INTERRUPTIBLE|TASK_FREEZABLE);
```

Comment:
This also needs to be guarded, TASK_FREEZABLE was introduced in kernel 6.1.

Reply:
Right, too bad I missed that :-(

Please feel free to set a PR that fixes it; else I will try to fix it, but this might take some days, since I am really busy at the moment..

Reply:
https://github.com/choff/anbox-modules/pull/8